ScnSoft-Education/LeetCodeTraining.Tests/TwoSumTests/TwoSumSolutionTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using LeetCodeTraining.TwoSum;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LeetCodeTraining.Tests.TwoSumTests
+{
+    public class TwoSumSolutionTests
+    {
+        [Fact]
+        public void TwoSum_ArrayWith2Nums()
+        {
+            //Arrange
+            var nums = new int[] { 3, 3 };
+            int target = 6;
+            var expectedResult = new int[] { 0, 1 };
+
+            //Act
+            var result = TwoSumSolution.TwoSum(nums, target);
+
+            //Assert
+            Assert.Equal(expectedResult.Length, result.Length);
+            Array.Sort(result);
```

Comment:
It seems this method should be moved to Act block.
For tether cases move please as well.